docs/development/maintenance.md

```diff
@@ -13,1292 +15,1464 @@ modules of ILIAS.
+    * Authority to Sign off Code Changes: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
+    * Authority to Sign off Conceptual Changes: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
+    * Authority to Sign off Testcase Changes: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
+    * Authority to (De-)Assign Authoritiees: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
 	* Tester: [TESTER MISSING](https://docu.ilias.de/goto_docu_pg_64423_4793.html)
+    * Unit-specific Guidelines, Rules, and Regulations: [LINK MISSING]('')
 
 [//]: # (END Tasks)
 
 [//]: # (BEGIN Taxonomy)
 
 * **Taxonomy**
-	* Maintenance Model: Classic
-	* 1st Maintainer: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
-	* 2nd Maintainer: N.A.
-	* Testcases: Tested separately in each module that supports taxonomies
+    * Authority to Sign off Code Changes: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
+    * Authority to Sign off Conceptual Changes: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
+    * Authority to Sign off Testcase Changes: Tested separately in each module that supports taxonomies
+    * Authority to (De-)Assign Authoritiees: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
 	* Tester: Tested separately in each module that supports taxonomies
+    * Unit-specific Guidelines, Rules, and Regulations: [LINK MISSING]('')
 
 [//]: # (END Taxonomy)
 
 [//]: # (BEGIN TermsOfServices)
 
 * **Terms of Services**
-	* Maintenance Model: Classic
-	* 1st Maintainer: [mjansen](https://docu.ilias.de/goto_docu_usr_8784.html)
-	* 2nd Maintainer: N.A.
-	* Testcases: Stefania Akgül (CaT)
+    * Authority to Sign off Code Changes: [mjansen](https://docu.ilias.de/goto_docu_usr_8784.html)
+    * Authority to Sign off Conceptual Changes: [mjansen](https://docu.ilias.de/goto_docu_usr_8784.html)
+    * Authority to Sign off Testcase Changes: Stefania Akgül (CaT)
+    * Authority to (De-)Assign Authoritiees: [mjansen](https://docu.ilias.de/goto_docu_usr_8784.html)
 	* Tester: Heinz Winter (CaT)
+    * Unit-specific Guidelines, Rules, and Regulations: [LINK MISSING]('')
 
 [//]: # (END TermsOfServices)
 
 [//]: # (BEGIN TestAndAssessment)
 
 * **Test & Assessment**
-	* Maintenance Model: Classic
-	* 1st Maintainer: [dstrassner](https://docu.ilias.de/goto_docu_usr_48931.html)
-	* 2nd Maintainer: [mbecker](https://docu.ilias.de/goto_docu_usr_27266.html)
+    * Authority to Sign off Code Changes: [mbecker](https://docu.ilias.de/goto_docu_usr_27266.html)
+        , [skergomard](https://docu.ilias.de/goto_docu_usr_44474.html)
+	* Authority to Sign off Conceptual Changes: [dstrassner](https://docu.ilias.de/goto_docu_usr_48931.html)
+    * Authority to Sign off Testcase Changes: [dstrassner](https://docu.ilias.de/goto_docu_usr_48931.html)
+    * Authority to (De-)Assign Authoritiees: [dstrassner](https://docu.ilias.de/goto_docu_usr_48931.html)
 	* Testcases: SIG E-Assessment
 	* Tester: Stefania Akgül (CaT), Stefanie Allmendinger (FAU)
```

Comment:
Please remove Stefania and Stefanie. Add Nadja Pfaffeicher, Kseniia Gribchenko, Testteam Uni Hohenheim and Testteam Kröpelin

[NDJ1508] --> Nicht freigegeben
[ksgrie]  --> https://docu.ilias.de/goto_docu_usr_95947.html